styleguide/derek/_partials/solutions/header-mediumIMAC.ejs

```diff
@@ -0,0 +1,17 @@
+<!--Header Medium-->
+<div class="imacBanner medium">
+  <div class="imacBanner-container">
+    <div class="imacBanner-textRow">
+      <div class="imacBanner-textContainer imacBanner-headlineMedium">
+        <h2 class="imacBanner-headline">the cloud is a mystery wrapped in a riddle.<br/>We can unwrap it for you</h2>
```

Comment:
These `<br />` tags can make the text wrapping look a little strange, depending on your browser size. Is this a requirement here?